sospice/catalog/tests/test_file_metadata.py

```diff
@@ -116,6 +116,13 @@ def test_get_observer(self, file_metadata):
         assert u.isclose(observer.lat, -4.2335761 * u.deg)
         assert u.isclose(observer.radius, 0.52286922 * u.au)
 
+    def test_file_mid_time(self, file_metadata):
+        fm = file_metadata
+        date_beg = fm.metadata["DATE-BEG"]
+        telapse = fm.metadata["TELAPSE"]
+        expected_mid_time = date_beg + pd.Timedelta(seconds=telapse / 2)
```

Comment:
Not sure whether the expected mid time should be obtained by the same formula as in the tested function, or it should simply be a constant.

Reply:
I think mid_range formula should remain the same, using a constant is a good option too.

I'll add a constant to resolve this